builder.go

```diff
@@ -67,7 +69,7 @@ func (logExecutor) Copy(excludes []string, copies ...Copy) error {
 }
 
 func (logExecutor) Run(run Run, config docker.Config) error {
-	log.Printf("RUN %v %t (%v)", run.Args, run.Shell, config.Env)
+	log.Printf("RUN %v %v %t (%v)", run.Args, run.Mounts, run.Shell, config.Env)
```

Comment:
This is probably more a question for @nalind and @rhatdan and I don't think it's a problem, but I don't know that it's not.  Any chance somebody might be parsing this log output already and having the mounts added in the middle rather than the end would mess that up?

Reply:
I would doubt people are scanning these log messages.

Reply:
I don't expect it to create problems, either.  We already tweaked the log format for COPY in #184, and I haven't heard anything yet.